manifests/setup/kustomization.yaml

```diff
@@ -1,12 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: kubesphere-logging-system
-commonLabels:
-  app.kubernetes.io/name: fluentbit-operator
-
 resources:
-- namespace-namespace.yaml
 - fluentbit-operator-clusterRoleBinding.yaml
 - fluentbit-operator-deployment.yaml
 - fluentbit-operator-crd.yaml
```

Comment:
The crd needn't be customized, I would suggest the resources includes below 2 files:

fluentbit-operator-deployment.yaml
fluentbit-operator-serviceAccount.yaml